src/OrchardCore/OrchardCore.Abstractions/Modules/Builder/OrchardCoreBuilder.cs

```diff
@@ -104,7 +125,7 @@ public OrchardCoreBuilder EnableFeature(string id)
                 {
                     var service = services[index];
                     if (service.ImplementationInstance is ShellFeature feature &&
-                        string.Equals(feature.Id, id, StringComparison.OrdinalIgnoreCase))
+                        String.Equals(feature.Id, id, StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't think we need `StringComparison.OrdinalIgnoreCase` here since `EnableFeature` is called from code not user input.

Reply:
Yes I was tempted to do it, just to not change the existing code here.

Hmm, anyway the provided id should be the exact feature id, otherwise it would not work.

Okay I will change it

Reply:
Done